content/en/cosign/public_deployment.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Public Deployment"
+category: "Cosign"
+position: 122
+---
+
+There is a public staging environment, or deployment, that is running Fulcio, Rekor and OIDC issuer.
+
+**NOTE** The staging environment provides neither SLO guarantees nor the same protection of the root key material for TUF. This environment is meant for development and testing only. It is not appropriate to use for production purposes.
+
+The endpoints are as follows:
+
+* https://fulcio.sigstage.dev
+* https://rekor.sigstage.dev
+* https://oauth2.sigstage.dev/auth
+
+These instances are operated and maintained in the same manner as the public production environment for Sigstore.
+
+### Usage
+
+To use this instance, follow the steps below:
+
+1. `rm -r ~/.sigstore`
+1. `curl -O https://raw.githubusercontent.com/sigstore/root-signing/main/staging/repository/1.root.json`
+1. `cosign initialize --mirror=https://tuf-repo-cdn.sigstore.dev --root=1.root.json`
+1. `COSIGN_EXPERIMENTAL=1 cosign sign --oidc-issuer "https://oauth2.sigstage.dev/auth" --fulcio-url "https://fulcio.sigstage.dev" --rekor-url "https://rekor.sigstage.dev" ${IMAGE_DIGEST}`
+1. `COSIGN_EXPERIMENTAL=1 cosign verify --rekor-url "https://rekor.sigstage.dev" ${IMAGE}`
```

Comment:
same and need to add the certificate flags as well

Reply:
added oidc issuer flags.